packages/sdk/src/models/swaps.ts

```diff
@@ -43,8 +43,8 @@ export default class Swap {
   }
 
   public async getSpotPrice(
-    inAsset: Asset,
-    outAsset: Asset,
+    inAsset: string | Asset,
```

Comment:
I'm still not convinced we need to support strings here, it should already be handled at higher level than this function.

Reply:
It is also.
But robustness principle.